<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN">
<html>
<head><meta name="generator" content="Pybtex">
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>Bibliography</title>
</head>
<body>
<dl>
<dt>1</dt>
<dd>Deren A.&nbsp;R. Eaton.
Toytree: <span class="bibtex-protected">A</span> minimalist tree visualization and manipulation library for <span class="bibtex-protected">Python</span>.
<em>Methods in Ecology and Evolution</em>, 11:187–191, 2020.
<a href="https://doi.org/10.1111/2041-210X.13313">doi:10.1111/2041-210X.13313</a>.</dd>
<dt>2</dt>
<dd>Diep&nbsp;Thi Hoang, Olga Chernomor, Arndt von Haeseler, Bui&nbsp;Quang Minh, and Le&nbsp;Sy Vinh.
<span class="bibtex-protected">UFBoot2: Improving the Ultrafast Bootstrap Approximation</span>.
<em>Molecular Biology and Evolution</em>, 35(2):518&ndash;522, 10 2017.
URL: <a href="https://doi.org/10.1093/molbev/msx281">https://doi.org/10.1093/molbev/msx281</a>, <a href="https://arxiv.org/abs/https://academic.oup.com/mbe/article-pdf/35/2/518/24367824/msx281.pdf">arXiv:https://academic.oup.com/mbe/article-pdf/35/2/518/24367824/msx281.pdf</a>, <a href="https://doi.org/10.1093/molbev/msx281">doi:10.1093/molbev/msx281</a>.</dd>
<dt>3</dt>
<dd>Bui&nbsp;Quang Minh, Heiko&nbsp;A Schmidt, Olga Chernomor, Dominik Schrempf, Michael&nbsp;D Woodhams, Arndt von Haeseler, and Robert Lanfear.
<span class="bibtex-protected">IQ-TREE 2: New Models and Efficient Methods for Phylogenetic Inference in the Genomic Era</span>.
<em>Molecular Biology and Evolution</em>, 37(5):1530&ndash;1534, 02 2020.
URL: <a href="https://doi.org/10.1093/molbev/msaa015">https://doi.org/10.1093/molbev/msaa015</a>, <a href="https://arxiv.org/abs/https://academic.oup.com/mbe/article-pdf/37/5/1530/33386032/msaa015.pdf">arXiv:https://academic.oup.com/mbe/article-pdf/37/5/1530/33386032/msaa015.pdf</a>, <a href="https://doi.org/10.1093/molbev/msaa015">doi:10.1093/molbev/msaa015</a>.</dd>
<dt>4</dt>
<dd>Remco Bouckaert, Timothy&nbsp;G. Vaughan, Joëlle Barido-Sottani, Sebastián Duchêne, Mathieu Fourment, Alexandra Gavryushkina, Joseph Heled, Graham Jones, Denise Kühnert, Nicola De&nbsp;Maio, Michael Matschiner, Fábio&nbsp;K. Mendes, Nicola&nbsp;F. Müller, Huw&nbsp;A. Ogilvie, Louis du&nbsp;Plessis, Alex Popinga, Andrew Rambaut, David Rasmussen, Igor Siveroni, Marc&nbsp;A. Suchard, Chieh-Hsi Wu, Dong Xie, Chi Zhang, Tanja Stadler, and Alexei&nbsp;J. Drummond.
<span class="bibtex-protected">BEAST</span> 2.5: <span class="bibtex-protected">An</span> advanced software platform for <span class="bibtex-protected">Bayesian</span> evolutionary analysis.
<em>PLOS Computational Biology</em>, 15(4):1–28, April 2019.
Publisher: Public Library of Science.
URL: <a href="https://doi.org/10.1371/journal.pcbi.1006650">https://doi.org/10.1371/journal.pcbi.1006650</a>, <a href="https://doi.org/10.1371/journal.pcbi.1006650">doi:10.1371/journal.pcbi.1006650</a>.</dd>
<dt>5</dt>
<dd>Kazutaka Katoh and Daron&nbsp;M. Standley.
<span class="bibtex-protected">MAFFT Multiple Sequence Alignment Software Version 7: Improvements in Performance and Usability</span>.
<em>Molecular Biology and Evolution</em>, 30(4):772&ndash;780, 01 2013.
URL: <a href="https://doi.org/10.1093/molbev/mst010">https://doi.org/10.1093/molbev/mst010</a>, <a href="https://arxiv.org/abs/https://academic.oup.com/mbe/article-pdf/30/4/772/6420419/mst010.pdf">arXiv:https://academic.oup.com/mbe/article-pdf/30/4/772/6420419/mst010.pdf</a>, <a href="https://doi.org/10.1093/molbev/mst010">doi:10.1093/molbev/mst010</a>.</dd>
<dt>6</dt>
<dd>Wytamma Wirth.
Dynamic beast.
August 2022.
URL: <a href="https://doi.org/10.5281/zenodo.7036457">https://doi.org/10.5281/zenodo.7036457</a>, <a href="https://doi.org/10.5281/zenodo.7036457">doi:10.5281/zenodo.7036457</a>.</dd>
<dt>7</dt>
<dd>Tim Vaughan.
Feast: tasty additions to beast 2.
2022.
URL: <a href="https://github.com/tgvaughan/feast">https://github.com/tgvaughan/feast</a>.</dd>
<dt>8</dt>
<dd>Jacob&nbsp;L Steenwyk, III Buida, Thomas&nbsp;J, Abigail&nbsp;L Labella, Yuanning Li, Xing-Xing Shen, and Antonis Rokas.
<span class="bibtex-protected">PhyKIT: a broadly applicable UNIX shell toolkit for processing and analyzing phylogenomic data</span>.
<em>Bioinformatics</em>, 37(16):2325&ndash;2331, 02 2021.
URL: <a href="https://doi.org/10.1093/bioinformatics/btab096">https://doi.org/10.1093/bioinformatics/btab096</a>, <a href="https://arxiv.org/abs/https://academic.oup.com/bioinformatics/article-pdf/37/16/2325/39948152/btab096.pdf">arXiv:https://academic.oup.com/bioinformatics/article-pdf/37/16/2325/39948152/btab096.pdf</a>, <a href="https://doi.org/10.1093/bioinformatics/btab096">doi:10.1093/bioinformatics/btab096</a>.</dd>
<dt>9</dt>
<dd>Jacob&nbsp;L. Steenwyk, Thomas&nbsp;J. Buida, Carla Gon<span class="bibtex-protected"><span class="bibtex-protected">ç</span></span>alves, Dayna&nbsp;C. Goltz, Grace Morales, Matthew&nbsp;E. Mead, Abigail&nbsp;L. LaBella, Christina&nbsp;M. Chavez, Jonathan&nbsp;E. Schmitz, Maria Hadjifrangiskou, Yuanning Li, and Antonis Rokas.
<span class="bibtex-protected">BioKIT</span>: a versatile toolkit for processing and analyzing diverse types of sequence data.
<em>biorxiv</em>, oct 2021.
URL: <a href="https://doi.org/10.1101%2F2021.10.02.462868">https://doi.org/10.1101%2F2021.10.02.462868</a>, <a href="https://doi.org/10.1101/2021.10.02.462868">doi:10.1101/2021.10.02.462868</a>.</dd>
</dl></body></html>
